app/controllers/application.js

```diff
@@ -16,8 +16,13 @@ export default class ApplicationController extends Controller {
 
   appTitle = 'Lokaal mandatenbeheer';
 
-  get isIndex() {
-    return this.router.currentRouteName === 'index';
+  get isNavigationVisible() {
```

Comment:
should this not live in the navigation bar component instead of passing it down as a prop?

Reply:
Im not always sure what the most explicit is, lmiting something in the component or seeing that it is limited when looking where the component is implemented.

Will move it in the component

Reply:
Moved it to the NavigationBar component